_pages/platform/pipeline.md

```diff
@@ -0,0 +1,86 @@
+
+With Algorithmia, pipelining machine learning is simple:
+
+- Algorithms are packaged as microservices with API endpoints: calling any algorithm or function is as easy as `algorithm.pipe(input)`
+- Pipelines can be input agnostic, since multiple languages and frameworks can be pipelined together 
+- You can set permissions for models and choose to allow a model to call other algorithms
+
+## Example ML Pipeline
+
+TODO: Graphic showing MLflow-based pipeline?
+
+### Create a Model
+
+We can use Algorithmia's [MLFlow integration](https://algorithmia.com/developers/clients/mlflow) to create our model. We'll use the example from the Algorithmia mlflow-algorithmia GitHub repository.
+
+First, install the mlflow-algorithmia package:
+
+```
+pip install mlflow-algorithmia
+```
+
+Next, clone the repository and build a model using the `sklearn_elasticnet_wine` example:
+
+
+```
+mlflow run examples/sklearn_elasticnet_wine/
+```
+
+This will create an mlruns directory that contains the trained model which we can then push to our ML repository on GitHub.
+
+### Automate model deployment
+
+There are a variety of ways to automate model deployment, such as integrating with a [CI/CD](https://algorithmia.com/developers/algorithm-development/ci-cd) system such as Jenkins or one built using GitHub actions. This allows us to automatically test our algorithms prior to deployment.
+
+Algorithmia provides a [GitHub action](https://github.com/marketplace/actions/deploy-to-algorithmia) we can use as an example. The GitHub action can be configured to automatically run a Jupyter notebook and generate a model for you, or if you already have a model checked in to your repository, the workflow can be configured with the existing model file path.
+
+```
+model_path:
+    description: 'Path of the model file to be uploaded to Algorithmia'
+    required: true
+    default: 'model.pkl'  
+```
+
+When the GitHub action runs, it will:
+
+- Take the ML model file from the configured path and upload it to your data collection at Algorithmia
+- Copy your inference (algorithm) files to your algorithm repository
+- Update/create a model_manifest.json file, connecting your inference (algorithm) code at Algorithmia with this newly uploaded model file
+
+The model manifest is a key feature for automating model deployments. It contains information about which GitHub repository and where the model file was uploaded from, what the MD5 hash of the model was when it was first created, the git commit SHA and commit message, and the time of the upload.
+
+By using this manifest, your inference script will know which model to load and use. It can also re-calculate the MD5 hash of the model file at inference time and compare it with the original MD5 hash that was calculated at the time of the upload, and make sure that the model file hasn't been changed.
```

Comment:
```suggestion
By using this manifest, your algorithm will know which model to load and use. It can also recalculate the MD5 hash of the model file at inference time and compare it with the original MD5 hash that was calculated at the time of the upload, and make sure that the model file hasn't been changed.
```